UPGRADE-6.0.md

```diff
@@ -172,7 +172,6 @@ Messenger
  * Removed the `prefetch_count` parameter in the AMQP bridge.
  * Removed the use of TLS option for Redis Bridge, use `rediss://127.0.0.1` instead of `redis://127.0.0.1?tls=1`
  * The `delete_after_ack` config option of the Redis transport now defaults to `true`
- * The `reset_on_message` config option now defaults to `true`
```

Comment:
Why removing this line? That's useful to know that the value can be cleared from the config.

Reply:
This line is already in FrameworkBundle:
> Make the framework.messenger.reset_on_message configuration option default to true

It doesn't belong to Messenger because configuration is related to FWB.